src/live_restore/live_restore_fs.c

```diff
@@ -281,7 +281,12 @@ __live_restore_fs_directory_list_worker(WT_FILE_SYSTEM *fs, WT_SESSION *wt_sessi
           lr_fs->os_file_system, wt_session, path_dest, prefix, &dirlist_dest, &num_dest_files));
 
         for (uint32_t i = 0; i < num_dest_files; ++i)
-            if (!WT_SUFFIX_MATCH(dirlist_dest[i], WTI_LIVE_RESTORE_STOP_FILE_SUFFIX)) {
+            /*
+             * The caller utilizes prefix to identify files necessary to it's module. Avoid
+             * returning live restore specific files to the caller.
```

Comment:
```suggestion
             * Live restore files should be invisible to the user. Skip them.
```

Reply:
Honestly it's pretty awful to implement, and suffixes feel nicer?

Reply:
Ignore that.